app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -23,6 +23,13 @@ class ManageIQ::Providers::Kubevirt::InfraManager < ManageIQ::Providers::InfraMa
   require_nested :Template
   require_nested :Vm
 
+  include ManageIQ::Providers::Kubernetes::VirtualizationManagerMixin
```

Comment:
this one adds dependency on the manageiq-providers-kubernetes project, therefore we add an explicit dependency on it